cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1060,6 +1082,20 @@ protected void jvmPerformanceOptions(List<EnvVar> envVars) {
             sbjAltNames.put("DNS.1", getServiceName());
             sbjAltNames.put("DNS.2", String.format("%s.%s.svc.%s", getServiceName(), namespace, KUBERNETES_SERVICE_DNS_DOMAIN));
             sbjAltNames.put("DNS.3", String.format("%s.%s.%s.svc.%s", podName.apply(cluster, i), getHeadlessServiceName(), namespace, KUBERNETES_SERVICE_DNS_DOMAIN));
+
+            int nextDnsId = 4;
+
+            // TODO: We need to refresh the certificates when the external addresses change!
```

Comment:
Eugh!

Reply:
I raised a separate issue for this - the impact it has is only hostname verification, not the exposing it self. I will try to get to it later ... perhaps over the weekend.